package-lock.json

```diff
@@ -8191,12 +8213,23 @@
                 "highlight.js": "11.8.0",
                 "lodash.debounce": "4.0.8",
                 "marked": "4.3.0"
+            },
+            "dependencies": {
+                "@prefecthq/vue-compositions": {
+                    "version": "1.5.9",
+                    "resolved": "https://registry.npmjs.org/@prefecthq/vue-compositions/-/vue-compositions-1.5.9.tgz",
+                    "integrity": "sha512-2W5bOxcLTvl+EjsEMyB7yj1Hp4ElebvHjlF0zHBYSlzSQZ6Bh3jkKoDjVVz8RfxkQ1+e25M/B7ZXjzV4T/NPNA==",
+                    "peer": true,
+                    "requires": {
+                        "jsdom": "^22.0.0"
+                    }
+                }
```

Comment:
🤨

Reply:
Oh this one is for prefect-design. 
<img width="1078" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/22418768/c2beced4-0455-4d3a-bfa2-f8262872b30e">

feel like that one's probably fine.